content/en/blog/2023/otel-go-metrics-sdk-stable.md

```diff
@@ -0,0 +1,18 @@
+---
+title: First stable release of OTel Go metric SDK
+linkTitle: Go Metric SDK is Stable
+date: 2023-09-29
+author: '[Fabrizio Ferri-Benedetti](https://github.com/theletterf/) (Splunk)'
+cSpell:ignore: Benedetti Fabrizio Ferri
+---
+
+The Go metric SDK has finally reached stable status with version 1.19.0 of
```

Comment:
Do we want to add a hyperlink to https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.19.0?